src/FsCheck/Gen.fs

```diff
@@ -313,6 +313,24 @@ module Gen =
     [<CompiledName("ListOf")>]
     let listOfLength n arb = sequence [ for _ in 1..n -> arb ]
 
+    ///Generates a random permutation of the given sequence using the Fisher-Yates shuffle algorithm.
```

Comment:
Isn't the specific algorithm used an implementation detail? Doesn't this documentation currently promise to always use that algorithm? What if, in the future, the need arises to switch to a different algorithm?